### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by ignatenkobrain** [TomasTomecek/pretty-git-prompt#49]
Just trying to figure out if I should keep it building / patching in Fedora or just retire and recommend people to use `starship` :)

**Comment by TomasTomecek** [comment]
Yes, it's alive and I'm using it daily, but I no longer do any development since it's feature complete for me.

Also my January and February were so crazy that I did not have any time to spend on my projects, that's why I'm replying right now.

I hope to look into Fedora builds and dependencies in coming days/weeks.

Thanks for taking care of it Fedora!
